projects/packages/videopress/src/videopress-divi/class-videopress-divi-extension.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Divi integration for Videopress.
+ *
+ * @package automattic/jetpack-videopress
+ **/
+
+use Automattic\Jetpack\Assets;
+
+/**
+ * Divi extension.
+ **/
+class VideoPress_Divi_Extension extends DiviExtension {
+
+	/**
+	 * The extension's js file namespace.
+	 *
+	 * @var string
+	 */
+	const JETPACK_VIDEOPRESS_DIVI_PKG_NAMESPACE = 'jetpack-videopress-divi-pkg';
+
+	/**
+	 * The gettext domain for the extension's translations.
+	 *
+	 * @since 1.0.0
```

Comment:
Will we version the extension differently, or could we use the package number with `$$next-version$$`?

Reply:
A subjective matter which I am not feeling strong about. Making the version be the package number sounds ok.